### PR TITLE
jbig2dec

### PR DESCRIPTION
**Pull request by scopatz**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by scopatz** [comment]
@conda-forge/staged-recipes - This is ready for review!

File: recipes/jbig2dec/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - libpng
+
+test:
+  commands:
+    - jbig2dec --help
+
+about:
+  home: https://jbig2dec.com/
+  license: AGPL-3.0-only
+  license_family: GPL
```
Comment (beckermr) [inline recipes/jbig2dec/meta.yaml]: Is this the right family? AGPL is pretty different from the rest of GPL. I am surprised anaconda included it there.

Comment (scopatz) [reply recipes/jbig2dec/meta.yaml]: Oh there is an AGPL.  I wish this list was on the conda-build website, it is impossible to find. https://github.com/conda/conda-build/blob/master/conda_build/license_family.py#L8

Comment (beckermr) [reply recipes/jbig2dec/meta.yaml]: cc @jjhelmus @forrestwaters

Comment (beckermr) [reply recipes/jbig2dec/meta.yaml]: Also the repo data patches will autogenerate this field in principle

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**